.github/workflows/main.yaml

```diff
@@ -15,9 +15,9 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 9
     steps:
-      - uses: actions/checkout@v3
-      - uses: ./.github/actions/setup-pnpm
+      - uses: actions/checkout@v4
+      - uses: oven-sh/setup-bun@v1
 
-      - run: pnpm install
-      - run: pnpm lint
-      - run: pnpm test
+      - run: bun install
+      - run: bun lint
+      - run: bun vitest
```

Comment:
can we make this work with `bun test`? think the happydom migration should be pretty straightforward

Reply:
Yup, been trying to migrate all to `bun test` but it seems a lot are not supported with happydom even. for instance, it fails whenever encountering svg files. related issue: https://github.com/oven-sh/bun/issues/3673
```
585 |         if (typeof window !== "undefined") {                                                                                                                                                                                                                       
586 |             let fullUrl;                                                                                                                                                                                                                                           
587 |             try {                                                                                                                                                                                                                                                  
588 |                 fullUrl = new URL(imgAttributes.src);                                                                                                                                                                                                              
589 |             } catch (e) {                                                                                                                                                                                                                                          
590 |                 fullUrl = new URL(imgAttributes.src, window.location.href);                                                                                                                                                                                        
                                      ^                                                                                                                                                                                                                                  
TypeError: "/_next/image?url=%2Fhome%2Fbongb%2Fflash%2Fsrc%2Fassets%2Fzadig_create_new_device.png&w=1200&q=75" cannot be parsed as a URL. 
```